content/syllabuses/bootcamp-web-dev-auto-boot.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Web development boot camp (with automarked katas)
+---
+
+- {{% contentlink path="projects/writing-assignment" flavour="markdown" %}}
+- {{% contentlink path="coding_aptitude_assessment/coding_challenges/task_1" flavour="javascript" %}}
+- {{% contentlink path="coding_aptitude_assessment/coding_challenges/task_2" flavour="javascript" %}}
+- {{% contentlink path="coding_aptitude_assessment/coding_challenges/task_3" flavour="javascript" %}}
+- {{% contentlink path="coding_aptitude_assessment/coding_challenges/task_4" flavour="javascript" %}}
+- {{% contentlink path="coding_aptitude_assessment/coding_challenges/task_5" flavour="javascript" %}}
+- {{% contentlink path="coding_aptitude_assessment/coding_challenges/task_6" flavour="javascript" %}}
+- {{% contentlink path="coding_aptitude_assessment/coding_challenges/task_7" flavour="javascript" %}}
+- {{% contentlink path="coding_aptitude_assessment/coding_challenges/task_8" flavour="javascript" %}}
+- {{% contentlink path="coding_aptitude_assessment/coding_challenges/task_9" flavour="javascript" %}}
+- {{% contentlink path="coding_aptitude_assessment/coding_challenges/task_10" flavour="javascript" %}}
+- {{% contentlink path="projects/git-exercises" %}}
+
+Things after this point are not strictly part of the bootcamp. It's extra stuff people can work on if they are ahead of the curve.
+
+- {{% contentlink path="topics/clean-code/java-script" %}}
```

Comment:
This is copied from the old bootcamp. I just skipped katas/level-0 because that stuff is covered by the challenge tasks above

Reply:
We could also leave this stuff out if it makes managing the boot camp easier.